2020-12-29 10:37:46.858 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 10:37:46.914 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 10:52:17.254 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 10:52:17.259 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 10:52:45.283 [http-nio-8011-exec-8] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.jdbc.BadSqlGrammarException: 
### Error querying database.  Cause: java.sql.SQLSyntaxErrorException: Unknown column 'gmt_tempend' in 'where clause'
### The error may exist in top/weidaboy/servicemain/mapper/TripMapper.java (best guess)
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### SQL: SELECT  id,name,userid,trip,tripreslove,is_delete,classid,majorid,collegeid,gmt_create,gmt_modified,gmt_tripend  FROM trip    WHERE  name = ? AND gmt_tempend LIKE ? ORDER BY gmt_tempend DESC
### Cause: java.sql.SQLSyntaxErrorException: Unknown column 'gmt_tempend' in 'where clause'
; bad SQL grammar []; nested exception is java.sql.SQLSyntaxErrorException: Unknown column 'gmt_tempend' in 'where clause']
2020-12-29 10:54:18.962 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 10:54:18.968 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 10:54:30.726 [http-nio-8011-exec-3] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.NullPointerException]
2020-12-29 10:56:14.425 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 10:56:14.429 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 10:56:37.046 [http-nio-8011-exec-8] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.NullPointerException]
2020-12-29 10:57:33.878 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 10:57:33.884 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 10:57:46.060 [http-nio-8011-exec-10] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.IllegalArgumentException: bound must be positive]
2020-12-29 10:59:08.695 [http-nio-8011-exec-7] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.NullPointerException]
2020-12-29 10:59:57.737 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 10:59:57.741 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:00:12.458 [http-nio-8011-exec-4] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.IllegalArgumentException: bound must be positive]
2020-12-29 11:07:08.667 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:07:08.671 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:08:06.841 [http-nio-8011-exec-3] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.NullPointerException]
2020-12-29 11:09:12.018 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:09:12.023 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:09:23.142 [http-nio-8011-exec-8] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.IllegalArgumentException: bound must be positive]
2020-12-29 11:09:30.090 [http-nio-8011-exec-1] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.NullPointerException]
2020-12-29 11:18:15.515 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:18:15.524 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:18:31.686 [http-nio-8011-exec-10] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.NullPointerException]
2020-12-29 11:20:58.674 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:20:58.678 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:21:31.699 [http-nio-8011-exec-3] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.NullPointerException]
2020-12-29 11:23:45.811 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:23:45.816 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:24:25.957 [http-nio-8011-exec-9] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.NullPointerException]
2020-12-29 11:25:39.604 [http-nio-8011-exec-4] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.IllegalArgumentException: bound must be positive]
2020-12-29 11:25:43.439 [http-nio-8011-exec-6] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.NullPointerException]
2020-12-29 11:25:54.705 [http-nio-8011-exec-7] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.NullPointerException]
2020-12-29 11:25:58.359 [http-nio-8011-exec-9] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.NullPointerException]
2020-12-29 11:27:13.495 [http-nio-8011-exec-1] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.NullPointerException]
2020-12-29 11:28:46.988 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:28:46.994 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:29:26.866 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:29:26.871 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:30:28.867 [restartedMain] WARN  o.s.b.w.s.c.AnnotationConfigServletWebServerApplicationContext - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'documentationPluginsBootstrapper' defined in URL [jar:file:/D:/MyJava/Mywarehouse/io/springfox/springfox-spring-web/2.7.0/springfox-spring-web-2.7.0.jar!/springfox/documentation/spring/web/plugins/DocumentationPluginsBootstrapper.class]: Unsatisfied dependency expressed through constructor parameter 1; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'webMvcRequestHandlerProvider' defined in URL [jar:file:/D:/MyJava/Mywarehouse/io/springfox/springfox-spring-web/2.7.0/springfox-spring-web-2.7.0.jar!/springfox/documentation/spring/web/plugins/WebMvcRequestHandlerProvider.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'requestMappingHandlerMapping' defined in class path resource [org/springframework/boot/autoconfigure/web/servlet/WebMvcAutoConfiguration$EnableWebMvcConfiguration.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Ambiguous mapping. Cannot map 'tripController' method 
top.weidaboy.servicemain.controller.TripController#getRandomTemp()
to {GET /servicemain/trip/getRandomTemp}: There is already 'tripController' bean method
top.weidaboy.servicemain.controller.TripController#test() mapped.
2020-12-29 11:31:25.670 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:31:25.674 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:31:49.659 [http-nio-8011-exec-10] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.IllegalArgumentException: bound must be positive]
2020-12-29 11:32:46.238 [http-nio-8011-exec-8] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.IllegalArgumentException: bound must be positive]
2020-12-29 11:34:09.270 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:34:09.275 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:34:24.905 [http-nio-8011-exec-4] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.IllegalArgumentException: bound must be positive]
2020-12-29 11:36:53.401 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:36:53.405 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:37:28.665 [http-nio-8011-exec-2] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.IllegalArgumentException: bound must be positive]
2020-12-29 11:39:44.364 [http-nio-8011-exec-8] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.NullPointerException]
2020-12-29 11:40:56.504 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:40:56.511 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:41:17.904 [http-nio-8011-exec-4] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.dao.DataIntegrityViolationException: 
### Error updating database.  Cause: java.sql.SQLException: Field 'gmt_create' doesn't have a default value
### The error may involve top.weidaboy.servicemain.mapper.TripMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO trip  ( id, name, userid, trip,   classid, majorid, collegeid )  VALUES  ( ?, ?, ?, ?,   ?, ?, ? )
### Cause: java.sql.SQLException: Field 'gmt_create' doesn't have a default value
; Field 'gmt_create' doesn't have a default value; nested exception is java.sql.SQLException: Field 'gmt_create' doesn't have a default value]
2020-12-29 11:43:01.083 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:43:01.087 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:43:16.844 [http-nio-8011-exec-7] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.IllegalArgumentException: bound must be positive]
2020-12-29 11:43:23.008 [http-nio-8011-exec-5] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.IllegalArgumentException: bound must be positive]
2020-12-29 11:43:27.789 [http-nio-8011-exec-2] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.IllegalArgumentException: bound must be positive]
2020-12-29 11:46:58.204 [http-nio-8011-exec-2] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.IllegalArgumentException: bound must be positive]
2020-12-29 11:49:23.793 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:49:23.797 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:49:52.668 [http-nio-8011-exec-5] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122911495256' for key 'PRIMARY'
### The error may involve top.weidaboy.servicemain.mapper.TempMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO temp  ( id, collegeid, majorid, classid, userid, name, temp,   gmt_create, gmt_modified, gmt_tempend )  VALUES  ( ?, ?, ?, ?, ?, ?, ?,   ?, ?, ? )
### Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122911495256' for key 'PRIMARY'
; Duplicate entry '2021122911495256' for key 'PRIMARY'; nested exception is java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122911495256' for key 'PRIMARY']
2020-12-29 11:50:53.682 [http-nio-8011-exec-4] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [java.lang.IllegalArgumentException: bound must be positive]
2020-12-29 11:54:23.979 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:54:23.983 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:57:39.380 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 11:57:39.384 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 12:00:09.980 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 12:00:09.984 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 12:00:57.000 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 12:00:57.004 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 12:01:12.340 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 12:01:12.347 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 12:03:03.531 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 12:03:03.538 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 12:03:49.441 [http-nio-8011-exec-10] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122912034964' for key 'PRIMARY'
### The error may involve top.weidaboy.servicemain.mapper.TripMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO trip  ( id, name, userid, trip,     collegeid, gmt_create, gmt_modified, gmt_tripend )  VALUES  ( ?, ?, ?, ?,     ?, ?, ?, ? )
### Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122912034964' for key 'PRIMARY'
; Duplicate entry '2021122912034964' for key 'PRIMARY'; nested exception is java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122912034964' for key 'PRIMARY']
2020-12-29 12:07:09.234 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 12:07:09.238 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 14:34:23.750 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 14:34:23.755 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 14:39:00.589 [http-nio-8011-exec-9] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122914390057' for key 'PRIMARY'
### The error may involve top.weidaboy.servicemain.mapper.TripMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO trip  ( id, name, userid, trip,   classid, majorid, collegeid, gmt_create, gmt_modified, gmt_tripend )  VALUES  ( ?, ?, ?, ?,   ?, ?, ?, ?, ?, ? )
### Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122914390057' for key 'PRIMARY'
; Duplicate entry '2021122914390057' for key 'PRIMARY'; nested exception is java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122914390057' for key 'PRIMARY']
2020-12-29 14:39:03.144 [http-nio-8011-exec-7] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122914390387' for key 'PRIMARY'
### The error may involve top.weidaboy.servicemain.mapper.TripMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO trip  ( id, name, userid, trip,     collegeid, gmt_create, gmt_modified, gmt_tripend )  VALUES  ( ?, ?, ?, ?,     ?, ?, ?, ? )
### Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122914390387' for key 'PRIMARY'
; Duplicate entry '2021122914390387' for key 'PRIMARY'; nested exception is java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122914390387' for key 'PRIMARY']
2020-12-29 14:41:19.167 [http-nio-8011-exec-7] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122914411991' for key 'PRIMARY'
### The error may involve top.weidaboy.servicemain.mapper.TripMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO trip  ( id, name, userid, trip,   classid, majorid, collegeid, gmt_create, gmt_modified, gmt_tripend )  VALUES  ( ?, ?, ?, ?,   ?, ?, ?, ?, ?, ? )
### Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122914411991' for key 'PRIMARY'
; Duplicate entry '2021122914411991' for key 'PRIMARY'; nested exception is java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122914411991' for key 'PRIMARY']
2020-12-29 15:18:23.865 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 15:18:23.870 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 15:25:06.982 [http-nio-8011-exec-5] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.http.converter.HttpMessageNotReadableException: Required request body is missing: public top.weidaboy.commonutils.R top.weidaboy.servicemain.controller.TripController.pagelist(java.lang.Long,java.lang.Long,java.lang.Long,top.weidaboy.servicemain.query.TripQuery)]
2020-12-29 15:25:21.132 [http-nio-8011-exec-4] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.http.converter.HttpMessageNotReadableException: Required request body is missing: public top.weidaboy.commonutils.R top.weidaboy.servicemain.controller.TripController.pagelist(java.lang.Long,java.lang.Long,java.lang.Long,top.weidaboy.servicemain.query.TripQuery)]
2020-12-29 15:26:13.830 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 15:26:13.836 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 16:07:13.745 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 16:07:13.753 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 16:32:08.807 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 16:32:08.812 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 16:32:51.681 [http-nio-8011-exec-10] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: com.baomidou.mybatisplus.core.exceptions.MybatisPlusException: Error: Method queryTotal execution error.
### The error may exist in file [D:\ADaDemo\graduation\service\service_main\target\classes\top\weidaboy\servicemain\mapper\xml\TripMapper.xml]
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### Cause: com.baomidou.mybatisplus.core.exceptions.MybatisPlusException: Error: Method queryTotal execution error.]
2020-12-29 16:50:26.621 [http-nio-8011-exec-3] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122916502651' for key 'PRIMARY'
### The error may involve top.weidaboy.servicemain.mapper.TripMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO trip  ( id, name, userid, trip,     collegeid, gmt_create, gmt_modified, gmt_tripend )  VALUES  ( ?, ?, ?, ?,     ?, ?, ?, ? )
### Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122916502651' for key 'PRIMARY'
; Duplicate entry '2021122916502651' for key 'PRIMARY'; nested exception is java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122916502651' for key 'PRIMARY']
2020-12-29 16:52:05.707 [http-nio-8011-exec-10] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122916520558' for key 'PRIMARY'
### The error may involve top.weidaboy.servicemain.mapper.TripMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO trip  ( id, name, userid, trip,   classid, majorid, collegeid, gmt_create, gmt_modified, gmt_tripend )  VALUES  ( ?, ?, ?, ?,   ?, ?, ?, ?, ?, ? )
### Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122916520558' for key 'PRIMARY'
; Duplicate entry '2021122916520558' for key 'PRIMARY'; nested exception is java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122916520558' for key 'PRIMARY']
2020-12-29 16:52:08.603 [http-nio-8011-exec-9] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.springframework.dao.DuplicateKeyException: 
### Error updating database.  Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122916520812' for key 'PRIMARY'
### The error may involve top.weidaboy.servicemain.mapper.TripMapper.insert-Inline
### The error occurred while setting parameters
### SQL: INSERT INTO trip  ( id, name, userid, trip,     collegeid, gmt_create, gmt_modified, gmt_tripend )  VALUES  ( ?, ?, ?, ?,     ?, ?, ?, ? )
### Cause: java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122916520812' for key 'PRIMARY'
; Duplicate entry '2021122916520812' for key 'PRIMARY'; nested exception is java.sql.SQLIntegrityConstraintViolationException: Duplicate entry '2021122916520812' for key 'PRIMARY']
2020-12-29 17:05:47.193 [http-nio-8011-exec-4] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: com.baomidou.mybatisplus.core.exceptions.MybatisPlusException: Error: Method queryTotal execution error.
### The error may exist in file [D:\ADaDemo\graduation\service\service_main\target\classes\top\weidaboy\servicemain\mapper\xml\TripMapper.xml]
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### Cause: com.baomidou.mybatisplus.core.exceptions.MybatisPlusException: Error: Method queryTotal execution error.]
2020-12-29 17:07:23.615 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 17:07:23.619 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 17:07:36.627 [http-nio-8011-exec-3] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: com.baomidou.mybatisplus.core.exceptions.MybatisPlusException: Error: Method queryTotal execution error.
### The error may exist in file [D:\ADaDemo\graduation\service\service_main\target\classes\top\weidaboy\servicemain\mapper\xml\TripMapper.xml]
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### Cause: com.baomidou.mybatisplus.core.exceptions.MybatisPlusException: Error: Method queryTotal execution error.]
2020-12-29 17:08:22.479 [http-nio-8011-exec-4] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: com.baomidou.mybatisplus.core.exceptions.MybatisPlusException: Error: Method queryTotal execution error.
### The error may exist in file [D:\ADaDemo\graduation\service\service_main\target\classes\top\weidaboy\servicemain\mapper\xml\TripMapper.xml]
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### Cause: com.baomidou.mybatisplus.core.exceptions.MybatisPlusException: Error: Method queryTotal execution error.]
2020-12-29 17:09:03.329 [http-nio-8011-exec-10] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: com.baomidou.mybatisplus.core.exceptions.MybatisPlusException: Error: Method queryTotal execution error.
### The error may exist in file [D:\ADaDemo\graduation\service\service_main\target\classes\top\weidaboy\servicemain\mapper\xml\TripMapper.xml]
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### Cause: com.baomidou.mybatisplus.core.exceptions.MybatisPlusException: Error: Method queryTotal execution error.]
2020-12-29 17:11:06.958 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 17:11:06.964 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 17:11:31.636 [http-nio-8011-exec-7] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: com.baomidou.mybatisplus.core.exceptions.MybatisPlusException: Error: Method queryTotal execution error.
### The error may exist in file [D:\ADaDemo\graduation\service\service_main\target\classes\top\weidaboy\servicemain\mapper\xml\TripMapper.xml]
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### Cause: com.baomidou.mybatisplus.core.exceptions.MybatisPlusException: Error: Method queryTotal execution error.]
2020-12-29 21:51:24.407 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 21:51:24.457 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 21:52:08.635 [http-nio-8011-exec-9] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: com.baomidou.mybatisplus.core.exceptions.MybatisPlusException: Error: Method queryTotal execution error.
### The error may exist in file [D:\ADaDemo\graduation\service\service_main\target\classes\top\weidaboy\servicemain\mapper\xml\TripMapper.xml]
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### Cause: com.baomidou.mybatisplus.core.exceptions.MybatisPlusException: Error: Method queryTotal execution error.]
2020-12-29 21:58:25.216 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 21:58:25.223 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 21:59:20.464 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 21:59:20.471 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 21:59:56.184 [http-nio-8011-exec-2] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: com.baomidou.mybatisplus.core.exceptions.MybatisPlusException: Error: Method queryTotal execution error.
### The error may exist in file [D:\ADaDemo\graduation\service\service_main\target\classes\top\weidaboy\servicemain\mapper\xml\TripMapper.xml]
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### Cause: com.baomidou.mybatisplus.core.exceptions.MybatisPlusException: Error: Method queryTotal execution error.]
2020-12-29 22:06:36.627 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 22:06:36.633 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 22:06:55.051 [http-nio-8011-exec-2] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: com.baomidou.mybatisplus.core.exceptions.MybatisPlusException: Error: Method queryTotal execution error.
### The error may exist in file [D:\ADaDemo\graduation\service\service_main\target\classes\top\weidaboy\servicemain\mapper\xml\TripMapper.xml]
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### Cause: com.baomidou.mybatisplus.core.exceptions.MybatisPlusException: Error: Method queryTotal execution error.]
2020-12-29 22:09:48.635 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 22:09:48.640 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 22:10:44.849 [http-nio-8011-exec-2] WARN  o.s.w.s.m.m.a.ExceptionHandlerExceptionResolver - Resolved [org.mybatis.spring.MyBatisSystemException: nested exception is org.apache.ibatis.exceptions.PersistenceException: 
### Error querying database.  Cause: com.baomidou.mybatisplus.core.exceptions.MybatisPlusException: Error: Method queryTotal execution error.
### The error may exist in file [D:\ADaDemo\graduation\service\service_main\target\classes\top\weidaboy\servicemain\mapper\xml\TripMapper.xml]
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### Cause: com.baomidou.mybatisplus.core.exceptions.MybatisPlusException: Error: Method queryTotal execution error.]
2020-12-29 22:13:08.048 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
2020-12-29 22:13:08.056 [restartedMain] WARN  com.netflix.config.sources.URLConfigurationSource - No URLs will be polled as dynamic configuration sources.
